beacon-chain/core/blocks/block_operations.go

```diff
@@ -774,6 +774,29 @@ func ProcessDeposits(ctx context.Context, beaconState *pb.BeaconState, body *eth
 	return beaconState, nil
 }
 
+// ProcessPreGenesisDeposit processes a deposit for the beacon state before chainstart.
+func ProcessPreGenesisDeposit(ctx context.Context, beaconState *pb.BeaconState,
+	deposit *ethpb.Deposit, valIndexMap map[[48]byte]int) (*pb.BeaconState, error) {
```

Comment:
It's better practice to avoid naming variables by their type, so instead of `valIndexMap` we can just have `validatorIndices map[[48]byte]int`

Reply:
ok changed